transformations/shuffle_within_segments/test.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
The method signature expects Lists but your test cases use strings.

Reply:
The problem is that tests for my transformation (ShuffleWithinSegments) pass successfully, tests fail for a different transformation (SuspectingParaphraser).
As for the signature, I have followed the example of some other TEXT_TAGGING transformers like https://github.com/GEM-benchmark/NL-Augmenter/tree/main/transformations/longer_location_ner

Reply:
The tests work now.